docs/mapping.md

```diff
@@ -1,33 +1,40 @@
+| Nested                            | `RECORD`                     | `STRUCT`               | Flat<sup>8</sup>                | `Entity`              | Group                             | `Message`               | Flat<sup>8</sup>    |
+| `Map[CharSequence, T]`            | `MAP[STRING, T]`             | -                      | -                               | -                     | -                                 | -                       |                     |
+| `Map[String, T]`                  | `MAP[STRING, T]`             | -                      | -                               | -                     | -                                 | -                       | -                   |
+| `java.time.Instant`               | `LONG`<sup>11</sup>          | `TIMESTAMP`            | -                               | `Timestamp`           | `LOGICAL[TIMESTAMP]`<sup>9</sup>  | -                       | -                   |
+| `java.time.LocalDateTime`         | `LONG`<sup>11</sup>          | `DATETIME`             | -                               | -                     | `LOGICAL[TIMESTAMP]`<sup>9</sup>  | -                       | -                   |
+| `java.time.OffsetTime`            | -                            | -                      | -                               | -                     | `LOGICAL[TIME]`<sup>9</sup>       | -                       | -                   |
+| `java.time.LocalTime`             | `LONG`<sup>11</sup>          | `TIME`                 | -                               | -                     | `LOGICAL[TIME]`<sup>9</sup>       | -                       | -                   |
+| `java.time.LocalDate`             | `INT`<sup>11</sup>           | `DATE`                 | -                               | -                     | `LOGICAL[DATE]`<sup>9</sup>       | -                       | -                   |
+| `org.joda.time.LocalDate`         | `INT`<sup>11</sup>           | -                      | -                               | -                     | -                                 | -                       | -                   |
+| `org.joda.time.DateTime`          | `INT`<sup>11</sup>           | -                      | -                               | -                     | -                                 | -                       | -                   |
+| `org.joda.time.LocalTime`         | `INT`<sup>11</sup>           | -                      | -                               | -                     | -                                 | -                       | -                   |
+ | `java.util.UUID`                  | `STRING`<sup>4</sup>         | -                      | ByteString (16 bytes)           | -                     | `FIXED[16]`<sup>9</sup>           | -                       | -                   |
```

Comment:
I'm not sure the 9. note holds for the UUID in parquet here

Reply:
good point! Removed it.